### DISCUSSION TITLE
Caddy as reverse proxy - X-Accel-Mapping

### DISCUSSION THREAD

**Discussion by ksampolski** [mastodon/mastodon#18585] [Q&A]
Welcome! I've been trying to use caddy as a reverse proxy for some time now, practically everything is working but I have a problem with X-Accel-Mapping header missing. Unfortunately it seems that I do not have enough skills to solve this problem myself, so I turn to you. My config is currently based on this: https://gist.github.com/yukimochi/bb7c90cbe628f216f821e835df1aeac1?permalink_comment_id=3607303#gistcomment-3607303. Can someone share his config which works without problems?

Thanx,
Sierran

**Comment by jbold** [comment]
I'm getting an error similar to this - did you ever find a fix? Navigation is broken after the first login to admin account management page - /auth/edit
